packages/create-svelte/package.json

```diff
@@ -2,24 +2,23 @@
 	"name": "create-svelte",
 	"version": "2.0.0-next.51",
 	"bin": "./bin.js",
+	"dependencies": {
+		"kleur": "^4.1.4",
+		"prompts": "^2.4.0"
+	},
 	"devDependencies": {
-		"@rollup/plugin-json": "^4.1.0",
 		"@sveltejs/kit": "workspace:*",
 		"@types/gitignore-parser": "^0.0.0",
 		"@types/prettier": "^2.2.3",
 		"gitignore-parser": "^0.0.2",
-		"kleur": "^4.1.4",
 		"prettier": "^2.2.1",
 		"prettier-plugin-svelte": "^2.2.0",
-		"prompts": "^2.4.0",
-		"rollup": "^2.41.1",
 		"sucrase": "^3.18.1",
 		"svelte-preprocess": "^4.7.0",
 		"tiny-glob": "^0.2.8"
 	},
 	"scripts": {
-		"dev": "rollup -cw",
-		"build": "node scripts/build-templates && rollup -c",
+		"build": "node scripts/build-templates",
```

Comment:
Now that the code is no longer completely contained inside `bin.js`, you need to update `files` below accordingly - `utils.js` needs to be added there.